### ISSUE TITLE
Possible to include default values and foreign keys?

### ISSUE THREAD

**Issue by mattias-sanfridsson** [TablePlus/tabledump#5]
Sorry for opening a bunch of issues here 😄

Is it possible to include the default values of columns? And also foreign keys? Would be awesome!

**Comment by huyphams** [comment]
Yep, it's possible because it's open source 😄I will support all those this weekend and update TablePlus plugin market.
In the meantime, you can take a look at this file: https://github.com/TablePlus/tabledump/blob/master/tabledump.tableplusplugin/library/helper.js
You can fix and install it, very easy.

**Comment by mattias-sanfridsson** [comment]
Right, I'll see if I find the time to open a PR. Thanks!
